test/metabase/query_processor/test_util.clj

```diff
@@ -487,18 +490,44 @@
                                [(:name table) (:name field)]))))
                    (t2/select-pks-set :model/Field :table_id [:in table-ids]))}))
 
+(defn- query-results [query]
```

Comment:
```suggestion
(defn- query-columns [query]
```
is this clearer? when I think of `results` I think of either the rows or the whole think returned from qp, not the columns metadata